### PR TITLE
Add Flasher v1.2.1

### PR DESCRIPTION
**Pull request by smirnov1993**
- added new version of the app
- added deb package installation instructions
- added list with links to other app versions

### CODE REVIEW

File: docs/flasher/gui.md [hunk 1]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
 
-* Download [Neutis Flasher for Linux](http://files.emlid.com/flash-tools/linux/neutis-manufacturing-flash-tool-v1.1.1.tar.bz2)
+!!! warning
+    Do not launch Neutis Manufacturing Flash Tool with **sudo** on Linux
+
+##### Deb package
+
+* Download [neutis-manufacturing-flash-tool_1.2.0_all.deb](http://files.emlid.com/flash-tools/linux/neutis-manufacturing-flash-tool_1.2.0_all.deb)
+
+* Using terminal navigate to directory with downloaded file and install it:
```
Comment (TatianaAndreeva) [inline docs/flasher/gui.md]: Using terminal navigate to the directory with the downloaded file and install it:

File: docs/flasher/gui.md [hunk 2]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
+
+        sudo dpkg -i neutis-manufacturing-flash-tool_1.2.0_all.deb
+
+* After installation is finished you can find the app in list of applications
```
Comment (TatianaAndreeva) [inline docs/flasher/gui.md]: After installation is finished you can find the app in the applications list

File: docs/flasher/gui.md [hunk 3]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
+
+<div style="text-align:center"><img src ="../../img/flasher/unity_dash.png" width=800></div>
+
+##### Tar archive
+
+* Download [neutis-manufacturing-flash-tool_1.2.0.tar.bz2](http://files.emlid.com/flash-tools/linux/neutis-manufacturing-flash-tool_1.2.0.tar.bz2)
+
 * Using terminal navigate to directory with downloaded archive and extract it:
```
Comment (TatianaAndreeva) [inline docs/flasher/gui.md]: Using terminal navigate to the directory with the downloaded archive and extract it:

File: docs/flasher/gui.md [hunk 4]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
+
+        cd ~/Downloads
+        tar -xvf neutis-manufacturing-flash-tool_1.2.0.tar.bz2
+        cd Neutis\ Manufacturing\ Flash\ Tool/
+
+* Run installation script to set up udev rules for Neutis:
+
+        sudo ./install.sh
+
+* Launch the application using starting script:
```
Comment (TatianaAndreeva) [inline docs/flasher/gui.md]: Launch the application using the starting script:

File: docs/flasher/gui.md [hunk 5]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
+</details>
 
 #### Windows
 
-* Download [Neutis Flasher for Windows](http://files.emlid.com/flash-tools/win/neutis-manufacturing-flash-tool-v1.1.1-setup.exe)
+* Download [neutis-manufacturing-flash-tool_1.2.0_setup.exe](http://files.emlid.com/flash-tools/win/neutis-manufacturing-flash-tool_1.2.0_setup.exe)
 * Launch downloaded executable and follow the installation wizard
-* You will be asked to install `Microsoft Visual C++ Redistributable`. Accept the licence agreement and install it
-* After installation is finished Neutis Manufacturing Flash Tool shortcut will appear on your desktop
+* After installation is finished Neutis Manufacturing Flash Tool shortcut will appear on your desktop and in Start menu
```
Comment (TatianaAndreeva) [inline docs/flasher/gui.md]: After installation is finished, Neutis Manufacturing Flash Tool shortcut will appear on your desktop and in the Start menu

File: docs/flasher/gui.md [hunk 6]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
 
 !!! note
     During the first launch of the app Windows may ask you for network access, bacause Flasher's inner architecture is based on client-server structure.
```
Comment (TatianaAndreeva) [inline docs/flasher/gui.md]: During the first app launching, Windows may ask the network access as Flasher's inner architecture is based on client-server structure. The application doesn't send any data over the Internet.

File: docs/flasher/gui.md [hunk 7]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
 
 #### Linux
 
-* Download [Neutis Flasher for Linux](http://files.emlid.com/flash-tools/linux/neutis-manufacturing-flash-tool-v1.1.1.tar.bz2)
+!!! warning
```
Comment (TatianaAndreeva) [inline docs/flasher/gui.md]: I'd add quotes symbol after the "warning" so we can see only the frame without a headline. In other docs we usually don't use headlines.

The same is for notes frames below.

File: docs/flasher/gui.md (hunk 2 above)
Comment (andrewyushkevich) [reply docs/flasher/gui.md]: After installation, you'll find the app in the applications list

File: docs/flasher/gui.md [hunk 8]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
- cd Neutis\ Manufacturing\ Flash\ Tool/
-```
-* Run starting script with `sudo`:
-```sh
-sudo ./neutis-flasher-start.sh
-```
+
+        cd ~/Downloads
+        tar -xvf neutis-manufacturing-flash-tool_1.2.0.tar.bz2
+        cd Neutis\ Manufacturing\ Flash\ Tool/
```
Comment (andrewyushkevich) [inline docs/flasher/gui.md]: Is it OK?

File: docs/flasher/gui.md [hunk 9]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
+
+* Run installation script to set up udev rules for Neutis:
```
Comment (andrewyushkevich) [inline docs/flasher/gui.md]: i think you should highlight **udev** or **udev rules**

File: docs/flasher/gui.md (hunk 4 above)
Comment (andrewyushkevich) [reply docs/flasher/gui.md]: i think you can leave here just

Launch the application

File: docs/flasher/gui.md [hunk 10]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
+
+        sudo ./install.sh
+
+* Launch the application using starting script:
+
+        ./neutis-flasher-start.sh
+
+<details close>
+<summary><font size="-1">Other versions</font></summary>
```
Comment (andrewyushkevich) [inline docs/flasher/gui.md]: Old versions

File: docs/flasher/gui.md [hunk 11]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
+* [neutis-manufacturing-flash-tool-v1.1.0.tar.bz2](http://files.emlid.com/flash-tools/linux/neutis-manufacturing-flash-tool-v1.1.0.tar.bz2)
+* [neutis-manufacturing-flash-tool-v1.0.0.tar.bz2](http://files.emlid.com/flash-tools/linux/neutis-manufacturing-flash-tool-v1.0.0.tar.bz2)
+
+</details>
 
 #### Windows
 
-* Download [Neutis Flasher for Windows](http://files.emlid.com/flash-tools/win/neutis-manufacturing-flash-tool-v1.1.1-setup.exe)
+* Download [neutis-manufacturing-flash-tool_1.2.0_setup.exe](http://files.emlid.com/flash-tools/win/neutis-manufacturing-flash-tool_1.2.0_setup.exe)
 * Launch downloaded executable and follow the installation wizard
```
Comment (andrewyushkevich) [inline docs/flasher/gui.md]: Launch it and follow the installation wizard
or
Launch .exe and follow the installation wizard

File: docs/flasher/gui.md (hunk 5 above)
Comment (andrewyushkevich) [reply docs/flasher/gui.md]: After installation, you'll find Neutis Manufacturing Flash Tool shortcut on your desktop and in the Start menu

File: docs/flasher/gui.md (hunk 6 above)
Comment (andrewyushkevich) [reply docs/flasher/gui.md]: During the first app launch, Windows may ask the network access as Flasher's inner architecture is based on client-server structure. The application doesn't send any data over the Internet.

File: docs/flasher/gui.md [hunk 12]
```diff
@@ -6,29 +6,64 @@ Neutis Manufacturing Flash Tool is available for Linux and Windows
...
-* You will be asked to install `Microsoft Visual C++ Redistributable`. Accept the licence agreement and install it
-* After installation is finished Neutis Manufacturing Flash Tool shortcut will appear on your desktop
+* After installation is finished Neutis Manufacturing Flash Tool shortcut will appear on your desktop and in Start menu
 
 !!! note
     During the first launch of the app Windows may ask you for network access, bacause Flasher's inner architecture is based on client-server structure.
     The application doesn't send any data over the Internet.
 
+<details close>
+<summary><font size="-1">Other versions</font></summary>
```
Comment (andrewyushkevich) [inline docs/flasher/gui.md]: Old versions

File: docs/flasher/gui.md (hunk 8 above)
Comment (smirnov1993) [reply docs/flasher/gui.md]: I've put quotes instead of escaping spaces with backslashes.

**Comment by smirnov1993** [comment]
I've made all the requested changes.
Version 1.2.0 won't be released as it has critical bug, so I close this PR and open new one for v.1.2.1 (#19)

**Pull request closed without merging**